types/openssi-websdk/index.d.ts

```diff
@@ -0,0 +1,1181 @@
+// Type definitions for openssi-websdk 0.0
+// Project: https://github.com/IBM-Blockchain-Identity/openssi-websdk#readme
+// Definitions by: Arne Rutjes <https://github.com/arner>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.1
+
+/**
+ * A simple logging mechanism for the SDK.
+ */
+// tslint:disable-next-line:no-unnecessary-class
+export class Logger {
```

Comment:
The only thing I see exported in the JS is `Agent`. This class, and all the functions you have exported here (`connect`, `disconnect`, etc.), don’t seem to exist in this file?